change/@fluentui-web-components-e7330a7e-73e6-4efd-a09f-84f488b8f917.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27506&commitId=56e5912f6747bdb0ad29a2592fc103e390b45df6&env=prod)

✅ There was 1 screenshots added, 0 screenshots removed, 1930 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Badge Converged.width constrained.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>